docs/cave/d2d-gis-shapefiles.md

```diff
@@ -36,26 +40,25 @@ Use the Apply button to save any changes you make to the default attribute setti
 
 ## Customizing the GIS Attribute Dialog
 
-You have the ability to highlight specific areas of the displayed map or map product image, and to hide other areas. These functionalities are executed through the **Attributes dialog**, commonly referred to as the "Attributes Table" because of its tabular style. To access the **Attributes dialog**, click and hold middle mouse button 3 (B3) on the Map ID in the Legend area to open the pop-up menu.
+You have the ability to highlight specific areas of the displayed map or map product image, and to hide other areas. These functionalities are executed through the **Attributes dialog**, commonly referred to as the "Attributes Table" because of its tabular style. To access the **Attributes dialog**, right click and hold mouse button 3 on the Map ID in the Legend area to open the pop-up menu.
 
 From the pop-up menu select the **Display Attributes** option to open the **Attributes** dialog.
 
 ![image](../images/gisattr.png)
 
 The Attributes dialog includes two menus, Annotation and Data, which allow you to control highlighting and visibility for both the displayed images and the information contained in the Attributes dialog. For each row of information on the Attributes dialog there is an associated map/map product image displayed on the Main Display Pane.
 
-
 ### Highlighting
 
 #### Highlighting Selected Areas
 
-To place a highlight around a selected area of the GIS display image and highlight the corresponding row on the Attributes dialog, click and hold B3 on the row you want to select. Check the Highlighted checkbox in the pop-up menu to activate highlighting. Multiple rows can be selected using Ctrl + B1 to select each row, which will highlight the selected row in blue. Then click and hold B3 on one of the selected rows; this will turn all selected blue rows dark gray  and open the pop-up menu. After checking the Highlighted checkbox all selected rows will be highlighted yellow and a highlighted border surrounding all the areas on the map corresponding to those selected rows will be highlighted with the color setup in the Preferences dialog box.
+To place a highlight around a selected area of the GIS display image and highlight the corresponding row on the Attributes dialog, right click and hold B3 on the row you want to select. Multiple rows can be selected using Ctrl + B1 to select each row, which will highlight the selected row in blue. Then click and hold B3 on one of the selected rows and check the Highlighted checkbox in the pop-up menu to activate highlighting. After checking the Highlighted checkbox all selected rows will be highlighted yellow and a highlighted border surrounding all the areas on the map corresponding to those selected rows will be highlighted with the color setup in the Preferences dialog box.
```

Comment:
"right click and hold B3"  -- do you right click (b2)? Or do you middle mouse click (b3)?

Reply:
also I think in general I would replace all instances of B1, B2, B3 with left-click, right-click, middle mouse click, respectively.  And maybe verify when it says to use B3, that it's actually B3 and not B2?